CityOfDetroit/bloom
# fix: add a11y statement

# Pull Request Template

## Issue Overview

This PR addresses #1583 


## Description

This PR adds the accessibility statement link to the footer and content to the new /accessibility page. Also, the title is a bit simpler than shown in the document but that decision was approved by product!

## How Can This Be Tested/Reviewed?

This can be tested from the homepage to ensure that the footer linking is correct and then on the /accessibility page to verify the content.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
